variablegroups/TEST-AKS-Common.json

```diff
@@ -0,0 +1,59 @@
+{
+  "variables": {
+    "Environment.name": {
+      "value": "test"
+    },
+    "Subscription.acrName": {
+      "value": "hmcts"
+    },
+    "Subscription.AgEmailReceivers": {
+      "value": "{\"receivers\":[{\"name\":\"rhollins\",\"emailAddress\":\"rafal.hollins@hmcts.net\"}]}"
```

Comment:
I wonder if this should go somewhere else.
ideally we don't want email addresses here, in case someone comes along and spams us. especially not individual ones...

Reply:
got rid of it for now, not needed for testing it.